scripts/release.sh

```diff
@@ -60,8 +60,18 @@ if [[ $(git status -uno --porcelain) ]]; then
   Die 'There are uncommitted changes.'
 fi
 
-# Make sure client is up to date with the latest changes.
-git checkout master
+if [[ "${SUFFIX}" = "lts" ]]; then
+  # LTS releases are based on N.0.x-lts branch, where N is the major release number.
+  # (Note that the minor version part of this BOM is always zero)
+  BASE_BRANCH=$(echo $VERSION | sed 's/\([0-9]\+\.[0-9]\+\)\.[0-9]\+/\1.x-lts/')
```

Comment:
So my understanding is that when it's LTS, the minor version is always 0, right?. Then I guess there's no need for the complex regex.
```suggestion
  BASE_BRANCH=$(echo $VERSION | sed 's/\([0-9]\+\)\.0\.[0-9]\+/\1.0.x-lts/')
```

If that's not the case, I think it's better to update the example to use `5.1.3` instead of `5.0.3`, which will give more clarity.

Reply:
Updated the regular expression.